apps/ett/app-ett/src/app/energy-track-and-trace-app.component.ts

```diff
@@ -34,15 +33,13 @@ const selector = 'ett-app';
       ${selector} {
         display: block;
         min-height: 100%;
+        background: var(--watt-color-neutral-grey-100);
       }
     `,
   ],
   template: `<router-outlet></router-outlet>`,
 })
-export class EnergyTrackAndTraceAppComponent {
-  @HostBinding('className')
-  className = 'mat-app-background';
-}
```

Comment:
Seems like removing this class has broken the layout on extra large screens
This branch:
![image](https://user-images.githubusercontent.com/6364586/150525808-551e0070-1347-43a5-855e-bf466f0ea63d.png)

Main branch:
![image](https://user-images.githubusercontent.com/6364586/150525884-f5bf1721-05f0-4b12-976a-b41738b4c97d.png)

`mat-app-background` probably does more than add a background color.